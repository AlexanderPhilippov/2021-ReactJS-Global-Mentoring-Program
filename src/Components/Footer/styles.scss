.footer {
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    height: 70px;
    display: flex;
    align-items: center;
    justify-content: center;
    background-color: #424242;

    &__text-logo {
        font-size: 25px;
        line-height: 25px;

        span {
            font-weight: bold;
        }

        color: #F65261;
    }
}