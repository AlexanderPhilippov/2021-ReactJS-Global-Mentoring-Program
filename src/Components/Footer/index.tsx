import React from 'react'
import './styles.scss'

const Footer: React.FC = () => {
    return(
        <div className="footer">
            <div className="footer__text-logo">
                <span>netflix</span>roulette
            </div>
        </div>
    )
}

export default Footer
