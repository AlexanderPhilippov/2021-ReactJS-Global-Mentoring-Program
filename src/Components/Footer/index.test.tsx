/**
 * @jest-environment jsdom
 */

import React from 'react'
import Footer from './index'
import { render } from '@testing-library/react'

it('Footer', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
})
