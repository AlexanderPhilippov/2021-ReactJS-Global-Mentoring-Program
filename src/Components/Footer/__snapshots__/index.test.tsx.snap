// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<div>
  <div
    class="footer"
  >
    <div
      class="footer__text-logo"
    >
      <span>
        netflix
      </span>
      roulette
    </div>
  </div>
</div>
`;
